tox.ini

```diff
@@ -79,7 +79,7 @@ setenv =
    TOX_INI_DIR = {toxinidir}
 
    ; Skip twistedchecker warnings that are not yet required in the required run
-   txchecker-travis-required: TWISTEDCHECKER_SKIP_WARNINGS = W9208,C0302
+   txchecker-travis-required: TWISTEDCHECKER_SKIP_WARNINGS = W9208,C0302,C0103,C9302
```

Comment:
This deactivates C0103 (invalid name) C9302 (invalid method name), because the tests fake OpenSSL APIs like `set_tmp_ecdh` that violate our coding standard.

These should be removed once https://github.com/twisted/twistedchecker/issues/120 and https://github.com/twisted/twistedchecker/issues/76 are closed.

Reply:
maybe add a FIXME comment here with reference to those tickets